### PR TITLE
IoTMiddlewareForFreeRTOS-XXXX: Initial Skytree specific telemetry changes

### PR DESCRIPTION
**Pull request by sagarepari**
Skytree specific changes for telemetry, more to be added soon.

Notes:
- I have forked the external repository. However, I have not added this under IoT-Embedded as we in initially thought. Because I wanted to avoid external repo under one more repo for avoiding any confusions.
- And after forking I have added the telemetry json payload construction logic.
- Also, you will find both my home and office Wi-Fi. I know you could now connect to my home WiFi 👍 :) 
- This is a minimum tested code and we trust MS :)

cc - @ErkanTree @HosseinAmirX

### CODE REVIEW

File: demos/projects/ST/b-l475e-iot01a/config/demo_config.h [hunk 1]
```diff
@@ -235,13 +255,23 @@ extern void vLoggingPrintf( const char * pcFormatString,
...
+#endif
 
 /**
  * @brief Wifi Password
  *
  */
-#define WIFI_PASSWORD                        "<Password>"
+// SAGAR: This is to allow me to work both at home and office.
+#ifdef HOME_SETUP
+    #define WIFI_PASSWORD                        "XYXMNAELRS65BH8A" // home
```
Comment (HosseinAmirX) [inline demos/projects/ST/b-l475e-iot01a/config/demo_config.h:271]: Is this safe?

File: demos/sample_azure_iot/sample_azure_iot.c [hunk 2]
```diff
@@ -78,26 +81,72 @@
...
+#define sampleazureiotTELEMETRY_OBJECT_UNIT2                      ( "unit2" )
+#define sampleazureiotTELEMETRY_OBJECT_UNIT3                      ( "unit3" )
+#define sampleazureiotTELEMETRY_OBJECT_PRESSURE                   ( "pressure" )
+#define sampleazureiotTELEMETRY_OBJECT_TEMPERATURE                ( "temperature" )
+
+// Telemetry
+#define sampleazureiotTELEMETRY_PRESSURE1                         ( "pressure1" )
+#define sampleazureiotTELEMETRY_PRESSURE2                         ( "pressure2" )
+#define sampleazureiotTELEMETRY_PRESSURE3                         ( "pressure3" )
+#define sampleazureiotTELEMETRY_MASS_FLOW                         ( "massflow" )
```
Comment (HosseinAmirX) [inline demos/sample_azure_iot/sample_azure_iot.c:107]: we may need 2 massflow values.

File: demos/sample_azure_iot/sample_azure_iot.c [hunk 3]
```diff
@@ -78,26 +81,72 @@
...
+#define sampleazureiotTELEMETRY_OBJECT_UNITS                      ( "units" )
+#define sampleazureiotTELEMETRY_OBJECT_EXTERNAL                   ( "environment" )
+#define sampleazureiotTELEMETRY_OBJECT_STATE                      ( "state" )
+#define sampleazureiotTELEMETRY_OBJECT_ERROR_STATE                ( "error-state" )
+// Secondary
+#define sampleazureiotTELEMETRY_OBJECT_UNIT1                      ( "unit1" )
+#define sampleazureiotTELEMETRY_OBJECT_UNIT2                      ( "unit2" )
+#define sampleazureiotTELEMETRY_OBJECT_UNIT3                      ( "unit3" )
+#define sampleazureiotTELEMETRY_OBJECT_PRESSURE                   ( "pressure" )
+#define sampleazureiotTELEMETRY_OBJECT_TEMPERATURE                ( "temperature" )
```
Comment (HosseinAmirX) [inline demos/sample_azure_iot/sample_azure_iot.c:101]: We may need 4 temperature values.

File: demos/sample_azure_iot/sample_azure_iot.c (hunk 2 above)
Comment (sagarepari) [reply demos/sample_azure_iot/sample_azure_iot.c:107]: Sure, we will make changes as we need.
This is as per the initial inputs. Let us meet again, including Viktor, after a few days once we are sure about the whole list.

File: demos/sample_azure_iot/sample_azure_iot.c (hunk 3 above)
Comment (sagarepari) [reply demos/sample_azure_iot/sample_azure_iot.c:101]: We have 4 temperature values if you look a bit down. And this is the object,

"temperature": {
                "cartridge1-1": "",
                "cartridge1-2": "",
                "cartridge2-1": "",
                "cartridge2-2": "",
                "cartridge3-1": "",
                "cartridge3-2": "",
                "cartridge4-1": "",
                "cartridge4-2": ""
            },

File: demos/projects/ST/b-l475e-iot01a/config/demo_config.h (hunk 1 above)
Comment (sagarepari) [reply demos/projects/ST/b-l475e-iot01a/config/demo_config.h:271]: it is not :)
As discussed, we will remove this once the code is completed. For pilots/ test, we will add this offline and will not be keeping in code. For now, keeping it for ease.

**Comment by HosseinAmirX** [comment]
PTTM @sagarepari  🥇

**Pull request merged**